synth-index/kaiko/priceAdapter.js

```diff
@@ -1,13 +1,14 @@
 const { Requester } = require('@chainlink/external-adapter')
 const Decimal = require('decimal.js')
+const { util } = require('@chainlink/ea-bootstrap')
 
 const getPriceData = async (synth) => {
   const url = `https://us.market-api.kaiko.io/v2/data/trades.v1/spot_exchange_rate/${synth.asset.toLowerCase()}/usd`
   const params = {
     sort: 'desc',
   }
   const headers = {
-    'X-Api-Key': process.env.API_KEY,
+    'X-Api-Key': util.pickRandomFromString(process.env.API_KEY, ","),
```

Comment:
Looks like `util.pickRandomFromString` fn is not available anymore, right? Should be `getRandomRequiredEnv`.